core/controllers/editor.py

```diff
@@ -39,6 +39,7 @@
 from core.domain import user_services
 from core.domain import value_generators_domain
 from core.platform import models
+from constants import CONSTANTS
```

Comment:
`sys.modules['constants'] = CONSTANTS` would set the constants module-level, but I don't think it's safe to set module dynamically and I didn't found any info about that either, so I think it's better to use from&import.